### PR TITLE
Upgrade lxml

### PR DESCRIPTION
**Pull request by nickmoreton**
# Upgrade lxml

Following a dependabot alter I have revised the version number for lxml.

Ticket URL: https://github.com/torchbox/wagtail-wordpress-import/security/dependabot/setup.py/lxml/open

In fact we had a fixed dependency in setup.py

---

- Testing
    - [x] CI passes
    - [x] These changes do not reduce test coverage
- Documentation.
    - [x] Documentation changes are not necessary because: there is no change for the developer to consider.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -41,7 +41,7 @@
     install_requires=[
         "Django>=3.1,<3.3",
         "Wagtail>=2.14,<2.16",
-        "lxml==4.6.3",
+        "lxml>=4.6,<4.7",
```
Comment (nimasmi) [inline setup.py]: **Issue**: This requirement still permits 4.6.3. We should insist on `>=4.6.5`.

**Question**: Why `<4.7`? Version 4.7.1 released yesterday doesn't claim to introduce any breaking changes. https://lxml.de/4.7/changes-4.7.1.html

**Review by nimasmi — CHANGES_REQUESTED** [review]

File: setup.py (hunk 1 above)
Comment (nickmoreton) [reply setup.py]: Thank Nick. How about if we do `"lxml>=4.7,<4.8"`

Comment (nimasmi) [reply setup.py]: Yep, that works for versions existing as we speak.

Comment (nickmoreton) [reply setup.py]: Thanks Nick i updated the version constraints.

**Review by Morsey187 — APPROVED** [review]
LGTM, tested locally!

**Pull request merged**